terraform/aks-cluster/main.tf

```diff
@@ -34,23 +33,18 @@ locals {
 }
 
 resource "azurerm_kubernetes_cluster" "adl_aks" {
```

Comment:
**[tfsec]** <[AVD-AZU-0041](https://aquasecurity.github.io/tfsec/v1.28.1/checks/azure/container/limit-authorized-ips/)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Cluster does not limit API access to specific IP addresses.